### DISCUSSION TITLE
.NET MAUI support

### DISCUSSION THREAD

**Discussion by imsam67** [shinyorg/shiny#1107] [Q&A] [answered]
I checked the documentation but I'm not 100% clear. Is .NET MAUI support available only in v3 Alpha? I don't see any mention of .NET MAUI in v2.6.x.

Am I understanding it correctly that for me to use this package in a .NET MAUI app, I need to install v3 Alpha?

**Comment by aritchie** [comment, marked as answer]
That's correct.  There is some minimal docs here: http://shinylib.net/release-notes/mobile/vnext.html

**Reply by imsam67** [reply]
Thank you!
